app/digitizingcontroller.h

```diff
@@ -68,6 +73,8 @@ private slots:
   QVector<QgsPoint> mRecordedPoints;  //!< for recording of linestrings
   QgsQuickAttributeModel *mRecordingModel = nullptr;  //!< to be used for highlight of feature being recorded
   QgsQuickMapSettings *mMapSettings = nullptr;
+  int mLineRecordingInterval = 3; // in seconds
+  QDateTime mLastTimeRecorded;
```

Comment:
I think we will also need to reset mLastTimeRecorded when starting recording, so if someone uses 30s interval, stops recording and then starts it again,  we would not wait and immediately record the first position.